resources/assets/ckeditor/plugins/iframedialog/plugin.js

```diff
@@ -0,0 +1,169 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
```

Comment:
@jebbdomingo Please make sure to change copyright docblocks

Reply:
@johanjanssens It's a third-party IFrame Dialog plugin https://ckeditor.com/cke4/addon/iframedialog

Reply:
@johanjanssens I'll be replacing it with magnificPopup anyway

Reply:
Oki.